main.py

```diff
@@ -98,15 +97,16 @@ def printlist(printer):
 
 @app.route('/reload/recent')
 def reload():
+    if not('last-fetch' in request.args and request.args.get('last-fetch').isdigit()):
+        return 'Invalid Request'
```

Comment:
```suggestion
        return 'Invalid Request', HTTP_400_BAD_REQUEST
```

Reply:
`HTTP_400_BAD_REQUEST` seems to be undefined. Resolved with usage of sending a status code of 400.

Reply:
ah, you would have to import static from flask and do static.HTTP...

just 400 is good though imo